src/main/java/com/qa/opencart/pages/ProductInfoPage.java

```diff
@@ -19,8 +19,10 @@ public class ProductInfoPage {
 	private By productImages = By.cssSelector("ul.thumbnails img");
 	private By productMetaData = By.xpath("(//div[@id='content']//ul[@class='list-unstyled'])[position()=1]/li");
 	private By productPriceData = By.xpath("(//div[@id='content']//ul[@class='list-unstyled'])[position()=2]/li");
-	private By quantity = By.id("input-quantity");
+	private By quantity = By.xpath("//input[@id='input-quantity']");
```

Comment:
give a meaningful name to the locator eg. productQuantity , prodQuantity

Reply:
done